build.gradle

```diff
@@ -335,3 +335,27 @@ if (project.hasProperty('javaLinkageArtifactIds')) {
     }
   }
 }
+
+if (project.hasProperty('runWithJava11')) {
+  allprojects {
+    plugins.withId('java') {
+
+      java {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+      }
+
+      tasks.withType(JavaCompile) {
+        options.with {
+          fork = true
+          forkOptions.javaHome = java8Home as File
+        }
+      }
+
+      test {
```

Comment:
Will this compile tests with Java11 as well?

Reply:
I'd also suggest to add test that validates runtime/compile time java version. This will make reviews much easier.

Reply:
@Ardagan regarding first comment -- it will compile tests with java 8.  TBH I thought that approach is the first we want to take. Possible next steps would be to compile/run with java 11. On the second thought, what would you like to achieve is to check if user's code compiled with J11 would run with beam (compiled with J8)? Let me know what you think

Reply:
@Ardagan 
Hey! I've added verification test to check bytecode and jvm.  It is launched only with `-PrunWithJava11` flag. I've already checked few jobs and did not noticed anything wrong. Let me know what do you think, thanks!